chef/cookbooks/ntp/templates/default/ntp.conf.erb

```diff
@@ -32,8 +35,11 @@ server <%= ntp_server %> iburst minpoll 4
 # up blocking replies from your own upstream servers.
 
 # By default, exchange time with everybody, but don't allow configuration.
-restrict -4 default kod notrap nomodify nopeer noquery
-restrict -6 default kod notrap nomodify nopeer noquery
+#restrict -4 default kod notrap nomodify nopeer noquery
+#restrict -6 default kod notrap nomodify nopeer noquery
+
+restrict -4 default ignore
+restrict -6 default ignore
```

Comment:
Might be good to have a comment for these, it's a little confusing to see the "By default" comment above, then the defaults commented out, and then no explanation for this addition.

Reply:
Yes I agree, a comment would be helpful here. Done